src/components/shapes/attributes.js

```diff
@@ -169,7 +169,28 @@ module.exports = templatedArray('shape', {
             'See `type` and `xsizemode` for more info.'
         ].join(' ')
     },
-
+    x0shift: {
+        valType: 'number',
+        dflt: 0,
+        min: -1,
+        max: 1,
+        editType: 'calc',
+        description: [
+            'Only relevant if `xref` is a (multi-)category axes. Shifts `x0` by a fraction of the',
+            'reference unit. E.g. -0.5 corresponds to the start of the category and 0.5 corresponds to the end of the category.'
+        ].join(' ')
```

Comment:
Does `(multi-) category` refer to one of the types as defined here: https://plotly.com/python/reference/layout/xaxis/#layout-xaxis-type

Is the `(multi-)`  to convey that it applies only to axes with the types "multicategory" or "category"

Reply:
Yes exactly, it applies to axes with `type` "multicategory" or `type` "category".